tests/bluetooth/shell/testcase.yaml

```diff
@@ -28,3 +28,86 @@ tests:
     extra_args: CONFIG_BT_PRIVACY=n
     platform_allow: native_posix
     tags: bluetooth
+
+# Bluetooth Audio Compile validation tests
```

Comment:
Isn't it missing tag value in all this tests?

Reply:
Probably. Let me add those in a PR

Reply:
@sylvioalves https://github.com/zephyrproject-rtos/zephyr/pull/37093